platform/o.n.bootstrap/launcher/windows/version.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#define COMPANY ""
+#define COMPONENT "NetBeans Platform Launcher"
+#define VER "101.1.0.0"
+#define FVER 101,1,0,0
```

Comment:
there might be more versions to change: https://issues.apache.org/jira/browse/NETBEANS-1590

Reply:
Yes, thanks, I noticed those earlier.  I've limited to the ones that @ebarboni updated in the other repo.  I assume that issue still applies.  We really need to find a way to centralize and configure these!

Reply:
maybe some of those don't need to be versions at all. Not sure what happens if the exe has just `Apache NetBeans` in the manifest.

Reply:
I'm not windows dev expert, I've no idea on how the manifest should be and what its quality should be to be win friendly. 

We may need to add Apache in front of NetBeans.

Reply:
looks like its required:
https://learn.microsoft.com/en-us/windows/win32/menurc/stringfileinfo-block

but its a string. So it could be "LauncherV1.1" or something arbitrary to not make the NB version connection.

Reply:
I've updated the version in the manifest files and added Apache in front of NetBeans in the component and name fields.

There are product and file versions in the stringfileinfo-block which appear to be required.  Usage of `BUILD_ID` also needs checking.  The version in the manifest files also appears to be required.  https://learn.microsoft.com/en-us/windows/win32/sbscs/application-manifests#assemblyIdentity  And that has a stricter format.  We'd also need to work out how to inject a version in there.

Hopefully this update is enough for now - we'll need to look at how we get pending changes in and release in time for NB21.